settings.gradle

```diff
@@ -0,0 +1,43 @@
+/*
+ * This file was generated by the Gradle 'init' task.
+ *
+ * The settings file is used to specify which projects to include in your build.
+ * For more detailed information on multi-project builds, please refer to https://docs.gradle.org/8.10.2/userguide/multi_project_builds.html in the Gradle documentation.
+ */
+
+pluginManagement {
+    plugins {
```

Comment:
I think here we need to add several other plugins as well. Can you check this repo[1]

[1] - https://github.com/ballerina-platform/module-ballerinax-aws.marketplace.mpe

Reply:
Added